ui/k-toolbar/main.kytos

```diff
@@ -41,6 +54,161 @@
+                    contentType: "application/json",
+                    data: JSON.stringify({"id": self.id,
+                                          "description": self.description,
+                                          "start": self.start,
+                                          "end": self.end,
+                                          "items": self.merged_list,         
+                                          "status": self.status,
+                    }),
+                url:this.$kytos_server_api + "kytos/maintenance",
+                success: function(data) {
+                    let notification = {
+                        icon: 'gear',
+                        title: 'Maintenance Window Created',
+                        description: 'Maintenance Window with id ' + self.id + ' was created.'
+                    }
+                    self.$kytos.$emit("setNotification", notification);
+                },
+                error: function(data) {
+                    let notification = {
+                        icon: 'gear',
+                        title: 'Maintenance Window not created',
+                        description: 'Maintenance Window with id ' + self.id + ' was not created.'
+                    }
+                   self.$kytos.$emit("setNotification", notification);
+               },
+             });
+             self.merged_list = [];
+            },
+            /*
+                Gets the Switch and Interface information using the GET api call from topology. 
+            */
+            loadNames: function() {
+                let _dpid_names = [];
+                let _interface_names = [];
+                var _this = this;
+
+                var request = $.ajax({
+                    url: this.$kytos_server_api + "kytos/topology/v3/switches",
+                    type: "GET",
+                    data: JSON.stringify(),
+                    dataType: "json",
+                    contentType: "application/json; charset=utf-8"
+               });
+
+               request.done(function(data) {
+                   let switches = data.switches;
+                   $.each(switches, function(i, sw) {
+                       if(sw.metadata.node_name){
+                           _dpid_names.push({"value": sw.id, "description": sw.metadata.node_name})
+                       }
+                       else{
+                           _dpid_names.push({"value": sw.id, "description": sw.id})
+                       }
+                       if(sw.interfaces){
+                           $.each(sw.interfaces, function(j, interface){
+                           if(Object.keys(interface.metadata).length == 0){
+                               _interface_names.push({"value": interface.id, "description": interface.id})
+                           }
+                           else{
+                               _interface_names.push({"value": interface.id, "description": interface.port_number})
```

Comment:
@rohanCodes001 in the same way that you've done for `switch.metadata.node_name`, it's desirable to have here the same for `interface.metadata.port_name` if it's defined, otherwise set the `id` in the description, could you update this accordingly instead of using the `port_number`?